redash/query_runner/cass.py

```diff
@@ -77,10 +94,14 @@ def run_query(self, query, user):
             if self.configuration.get('username', '') and self.configuration.get('password', ''):
                 auth_provider = PlainTextAuthProvider(username='{}'.format(self.configuration.get('username', '')),
                                                       password='{}'.format(self.configuration.get('password', '')))
-                connection = Cluster([self.configuration.get('host', '')], auth_provider=auth_provider, protocol_version=3)
+                connection = Cluster([self.configuration.get('host', '')],
+                                     auth_provider=auth_provider,
+                                     protocol_version=self.configuration.get('protocol', ''),
+                                     cql_version=self.configuration.get('cqlversion', ''))
```

Comment:
I had misadventures with those proto and cql versions, we have several production versions and even in the official cqlsh client we should juggle with these params, it is a mess, they fixed the version in the code, it tells you what version to use but does not change it... 👍 
```
cat /usr/local/bin/cqlsh | grep -i DEFAULT_CQLVER
DEFAULT_CQLVER = '3.3.1'
parser.add_option('--cqlversion', default=DEFAULT_CQLVER,
                 cqlver=DEFAULT_CQLVER, keyspace=None
```
```
cqlsh 127.0.0.1 -u cassandra
Password:
Connection error: ('Unable to connect to any servers', {'127.0.0.1': ProtocolError("cql_version '3.3.1' is not supported by remote (w/ native protocol). Supported versions: [u'3.4.0']",)})
```
I don't know the python client, I will do some tests before editing these.

Anyway the default value to "None" should be a good choice.